tfrobot/pkg/deployer/load.go

```diff
@@ -247,3 +226,32 @@ func loadDeploymentWithContractID(ctx context.Context, tfPluginClient deployer.T
 
 	return workloads.NewDeploymentFromZosDeployment(dl, nodeID)
 }
+
+func getContractsWithProjectName(ctx context.Context, tfPluginClient deployer.TFPluginClient, nodeGroup string) ([]uint64, error) {
+	// try to load group with the new name format "vm/<group name>"
+	name := fmt.Sprintf("vm/%s", nodeGroup)
+
+	contracts, err := tfPluginClient.ContractsGetter.ListContractsOfProjectName(name, true)
+	if err != nil || len(contracts.NodeContracts) == 0 {
```

Comment:
if there is an error it should return immediately, right? that's different than finding no contracts.